### PR TITLE
Upgraded to Akka.NET v1.5.0-beta1

### PR DESCRIPTION
**Pull request by Aaronontheweb**
## Changes

Had to rewrite the guts of the `LoggerFactoryLogger` - but it's a lot simpler now and more consistent with the rest of Akka.NET's loggers.

We'll need to update the Akka.Hosting.LoggingDemo to re-include Serilog support once https://github.com/akkadotnet/Akka.Logger.Serilog is also updated to support v1.5.0-beta1 or later.

## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).
* [x] Changes in public API reviewed, if any.

### CODE REVIEW

File: src/Akka.Hosting.API.Tests/verify/CoreApiSpec.ApproveCore.verified.txt [hunk 1]
```diff
@@ -185,7 +185,6 @@ namespace Akka.Hosting.Logging
 {
     public class LoggerFactoryLogger : Akka.Actor.ActorBase, Akka.Dispatch.IRequiresMessageQueue<Akka.Event.ILoggerMessageQueueSemantics>
     {
-        public const string DefaultTimeStampFormat = "yy/MM/dd-HH:mm:ss.ffff";
```
Comment (Aaronontheweb) [inline src/Akka.Hosting.API.Tests/verify/CoreApiSpec.ApproveCore.verified.txt:188]: No longer use this format string - we rely on Akka.NET's internals instead.

File: src/Akka.Hosting/Logging/LoggerFactoryLogger.cs [hunk 2]
```diff
@@ -68,71 +62,19 @@ protected override bool Receive(object message)
...
-                    default:
-                        sb.Append($"Failed to invoke {obj.GetType()}.ToString().");
-                        break;
-                }
-
-                sb.AppendLine(" Please take a look at the logging call where this occurred and fix your format string.");
-                sb.Append(ex);
-                return sb.ToString();
-            }
+            _akkaLogger.Log<LogEvent>(GetLogLevel(log.LogLevel()), new EventId(), log, log.Cause, (@event, exception) => @event.ToString());
```
Comment (Aaronontheweb) [inline src/Akka.Hosting/Logging/LoggerFactoryLogger.cs:65]: We have a minimalist logging call now - it defers any type of formatting until the Microsoft.Extensions.Logging pipeline calls it. Uses the built-in `LogEvent.ToString` method that the rest of Akka.NET's loggers do.

File: src/Examples/Akka.Hosting.LoggingDemo/Program.cs [hunk 3]
```diff
@@ -39,7 +39,7 @@
             setup.AddLoggerFactory();
             
             // Example: Adding a serilog logger
-            setup.AddLogger<SerilogLogger>();
+            //setup.AddLogger<SerilogLogger>(); // TODO: add back once upgraded to v1.5 APIs
```
Comment (Aaronontheweb) [inline src/Examples/Akka.Hosting.LoggingDemo/Program.cs:42]: Need to add this back once the Serilog plugin has been updated.

**Review by Aaronontheweb — COMMENTED** [review]
Reviewed my changes

**Review by Arkatufus — APPROVED** [review]
LGTM

**Pull request merged**